modules/ixBidAdapter.js

```diff
@@ -238,26 +272,78 @@ function isValidSize(size) {
  * @return {boolean}          True if the size object is an element of the size array, and false
  *                            otherwise.
  */
-function includesSize(sizeArray, size) {
+function includesSize(sizeArray = [], size) {
   if (isValidSize(sizeArray)) {
     return sizeArray[0] === size[0] && sizeArray[1] === size[1];
   }
-
   for (let i = 0; i < sizeArray.length; i++) {
     if (sizeArray[i][0] === size[0] && sizeArray[i][1] === size[1]) {
       return true;
     }
   }
+  return false;
+}
+
+/**
+ * Checks if all required video params are present
+ * @param {object} bid               Bid Object
+ * @param {object} mediaTypeVideoRef Ad unit level mediaTypes object
+ * @param {object} paramsVideoRef    IX bidder params level video object
+ * @returns bool                     Are the required video params available
+ */
+function checkVideoParams(bid, mediaTypeVideoRef, paramsVideoRef) {
+  let reqParamsPresent = true;
+
+  if (!mediaTypeVideoRef) {
+    utils.logWarn('IX Bid Adapter: mediaTypes.video is the preferred location for video params in ad unit');
+  }
+
+  for (let property of REQUIRED_VIDEO_PARAMS) {
+    const propInMediaType = mediaTypeVideoRef && mediaTypeVideoRef.hasOwnProperty(property);
+    const propInVideoRef = paramsVideoRef && paramsVideoRef.hasOwnProperty(property);
+
+    if (!propInMediaType && !propInVideoRef) {
+      utils.logError('IX Bid Adapter: ' + property + ' is not included in either the adunit or params level');
```

Comment:
@punkiller (or @lksharma and @umakajan for OOO backup) - Could this line be updated to a logWarn? 

This pr duplicates this code execution in `buildRequests`  where previously it was only in `isBidRequestValid`, the latter makes sense for an error since no request would go out. However when it is run within `buildRequests` there are instances where we are seeing errors for your adapter but requests are still made. 

Maybe a better suggestion would be to move the logging to the `buildRequests` area instead of in this function?

Reply:
If errors should only be logged when no request is going out then yes it can be changed to a warn.
Yes this logging can be moved to buildRequests and changed to a warn.
Whatever the result, it must be tested with an integration example containing MultiFormat Ad unit that has missing required video properties.
So the request for banner ads will still go out, but video request will not go out and only a warn is logged.

Reply:
@ix-prebid-support -- for visibility to the thread

Reply:
@ncolletti would you be able to provide us with a test page where our team can debug this issue?

We are not able to reproduce this behaviour at this moment